packages/plus/src/components/Navigation/components/Item.tsx

```diff
@@ -590,6 +588,7 @@ export const Item = memo(
                   }
                   animation={animation}
                   disabled={disabled}
+                  whiteSpace="pre-wrap"
```

Comment:
Here it's no going to work as before: `white-space: ${({ animation }) => (animation ? 'nowrap' : 'normal')};` its conditioned by animation

Reply:
Previously there was `white-space: pre-wrap` which took over `white-space: ${({ animation }) => (animation ? 'nowrap' : 'normal')};` (see l131 & l136) , that's why I put pre-wrap here